### PR TITLE
integrate radiation field

### PR DESCRIPTION
**Pull request by wkerzendorf**
New integration of radiation field

### CODE REVIEW

File: tardis/radiation_field/base.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        self.dilution_factor = dilution_factor
+        self.t_rad = self.t_radiative
+        self.w = self.dilution_factor
+        self.opacities = opacities
+        self.source_function = packet_source
+
+
+def convert_config_to_radiationfield_state(
+    config, no_of_shells, temperature=None
+):  ### make it a to_method in the end
```
Comment (andrewfullard) [inline tardis/radiation_field/base.py:62]: Good because otherwise this is really clunky!

File: tardis/radiation_field/base.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import numpy as np
+from astropy import units as u
+
+from tardis.montecarlo.packet_source import BasePacketSource
+from tardis.montecarlo.montecarlo_numba.numba_interface import OpacityState
+
+
+class RadiationField:
```
Comment (andrewfullard) [inline tardis/radiation_field/base.py:8]: Do we want to be clear about this being a radiationfield*state*?

File: tardis/radiation_field/base.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        opacities: OpacityState,
+        packet_source: BasePacketSource,
+    ):
+        self.t_radiative = t_radiative
+        self.dilution_factor = dilution_factor
+        self.opacities = opacities
+        self.packet_source = packet_source
+
+
+class MonteCarloRadiationFieldState:
```
Comment (andrewfullard) [inline tardis/radiation_field/base.py:36]: Should this inherit from the above class to reduce code duplication? Is it even necessary? There is duplication of t_rad.

File: tardis/radiation_field/base.py (hunk 2 above)
Comment (wkerzendorf) [reply tardis/radiation_field/base.py:8]: sure

File: tardis/radiation_field/base.py (hunk 3 above)
Comment (wkerzendorf) [reply tardis/radiation_field/base.py:36]: I just copied from the previous implementation. But ultimately, we could also only have that and then add a superclass later.

**Comment by wkerzendorf** [comment]
@andrewfullard I've included your suggestions.

**Review by andrewfullard — APPROVED** [review]

**Comment by andrewfullard** [comment]
Looks like `black` and tests are needed

**Pull request merged**